app/validation/validators.py

```diff
@@ -0,0 +1,129 @@
+from datetime import datetime
+
+from app.validation.error_messages import error_messages
+
+from wtforms import validators
+
+
+class IntegerCheck(object):
```

Comment:
Doesn't WTForms provide these kinds of validators out the box; possibly my not understanding how this integrates with what we already have.